### PR TITLE
chore(showcase): Add Yoseph.tech

### PR DESCRIPTION
**Pull request by radding**
## Description
Added two new sites to `docs/sites.yml`: my personal blog and an eCommerce site I built for a client.

### CODE REVIEW

File: docs/sites.yml [hunk 1]
```diff
@@ -8323,6 +8323,31 @@
...
+  featured: false
+- title: Gildform
+  main_url: https://www.gildform.com
+  url: https://www.gildform.com
+  source_url:
+  description: >
+    Gildform is an on demand jewelry manaufacturer, taking 3D models uploaded through the site and converting it to a real piece of jewelry
+  categories:
+    - eCommerce
+    - Manufacturing
```
Comment (pieh) [inline docs/sites.yml]: We don't have `Manufactoring` in our categories whitelist - see for list of available ones https://github.com/gatsbyjs/gatsby/blob/master/docs/categories.yml#L64-L118
```suggestion
```

File: docs/sites.yml [hunk 2]
```diff
@@ -8323,6 +8323,31 @@
...
+    - Technology
+    - Web Development
+    - Open Source
+  built_by: Yoseph Radding
+  built_by_url: https://github.com/radding
+  featured: false
+- title: Gildform
+  main_url: https://www.gildform.com
+  url: https://www.gildform.com
+  source_url:
```
Comment (pieh) [inline docs/sites.yml]: Please remove the field if there is no `source_url`:
```suggestion
```

**Comment by radding** [comment]
> The GlidForm site is very weird - it doesn't use `gatsby-image`, it loads some scripts in render-blocking way, some links doesn't seem to use `gatsby-link`. Can you look into this stuff? There are many things to improve performance of the site as right now lighthouse audit give it pretty bad score (for gatsby site)

Unfortunately, I don't have control over that site anymore so I can't make any changes or updates to it :disappointed:

**Comment by pieh** [comment]
> Unfortunately, I don't have control over that site anymore so I can't make any changes or updates to it 😞

In that case, I would not add that site to showcase - it doesn't follow our best practices (not the way we recommend to build sites).

Your personal blog is ok, so would just add this one.

**Comment by radding** [comment]
> > Unfortunately, I don't have control over that site anymore so I can't make any changes or updates to it disappointed
> 
> In that case, I would not add that site to showcase - it doesn't follow our best practices (not the way we recommend to build sites).
> 
> Your personal blog is ok, so would just add this one.

Fixed this btw

**Pull request merged**